### PR TITLE
Completed the Capstone project with all requirements implement

### PR DESCRIPTION
**Pull request by bobb-Rob**
## The following was implemented in this project.

- [ ] Personalize the content of the project. 
- [ ] follow the  design guidelines including:
    - Colors.
    - Typography: font face, size, and weight.
    - Layout: composition and space between elements.
- [ ] Built the two pages home and about with version for mobile: up to 768px wide and desktop: 768px or wider.
- [ ] Interactions - logo, hamburger button, and links in the menu work as expected. The logo in the header links to the home page.
- [ ] Featured artists' section created dynamically in JavaScript.

***I included a video description of the project in the README***

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Special thanks to [Creative Commons](https://creativecommons.org/licenses/by-nc/4.0/) for the excellent Design guide.
```
Comment (danmainah) [inline README.md]: - [x] Kindly rewrite your acknowledgment to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), currently, you have not given the author credit.

File: README.md [hunk 2]
```diff
@@ -1 +1,86 @@
...
+
+> This project is the Final capstone project of the first module at microverse. It is aimed at building a responsive UI of a website for an event themed "_**Global Drummers Festival 2022**_". 
+
+In this project, I built the home and the about page of the website and it is adaptable to all screen sizes. The template I used as a guide was originally designed by [Cindy Shin in Behance](https://www.behance.net/adagio07).
+
+## Live Demo
+Below is the link to the demo of this porject.
+
+> [DEMO](https://bobb-rob.github.io/Global-Drummers-Workshop/)
+
```
Comment (danmainah) [inline README.md:14]: - [ ] Kindly check on your live demo, the  page breaks when I click the logo also  the home page is not showing images 
image

File: html/about.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+<body class="lato-font">
+    <header id="nav-container">
+        <a class="logo-container" href="../index.html">
+            <img class="logo" src="../assets/icons/Drum-festival-logo.svg" alt="Drum Festival Logo">
+        </a>
+        <nav class="navbar">
+            <ul class="navlinks-wrapper">
+                <li>
+                    <a href="./about.html">About</a>
```
Comment (danmainah) [inline html/about.html]: - [x] Kindly change this link and text, to a link to the homepage and a text `home`, you are repeating yourself providing a link to the about page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                <li>
+                    <a href="#performing-artist">News</a>
+                </li>
+            </ul>
+            <button type="button" class="nav-join-btn">
+                Join Drum Festival
+            </button>
+            <div class="hamburger-btn">
+                <img class="hamburger" src="./assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
```
Comment (danmainah) [inline index.html:68]: - [x] Kindly make sure your hamburger menu is on the left side, currently its on the right , also remove the logo to match the design given.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3da71c9faf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style/main.css">
+    <script src="./javascript/main.js" defer></script>
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="lato-font">
+    <header id="nav-container">
```
Comment (danmainah) [inline index.html:20]: - [x] Kindly add the  top short bar section with social media links, logout, and language links, check the design below 
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+        <div class="orange-red lato-font">"Hello! Drummers!!"</div>
+        <h1 class="headline-main-text orange-red cocogoose-font">GLOBAL DRUM FESTIVAL & WORKSHOP 2022</h1>
+        <p class="black-like lato-font home-headline-body">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Quae, temporibus?
+            Aliquid reiciendis adipisci error voluptatibus illo quos unde, tempore, voluptate porro id velit. Ea
+            voluptatibus maiores nemo!</p>
+        <h3 class="black-like cocogoose-font event-date">15.09.2022(THU) ~ 17(SAT)</h3>
+        <p class="black-like lato-font venue">@ O2 Arena, London, United Kingdom</p>
+    </section>
```
Comment (danmainah) [inline index.html]: - [x] Kindly make sure your design is consistent in all page widths, currently, it shifts depending on width, making it hard to determine your intended design. Here is a [link](https://www.loom.com/share/0814161ed120414caf484e0ed0fa962c) to the video showing the change.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+        </section>
+        <section id="performing-artist" class="performing-artists-section">
+            <div class="perform-header">
+                <h4>Performing Drummers</h4>
+                <div class="thin-line">
+                </div>
+            </div>
+            <ul class="performing-artists">
+
+            </ul>
```
Comment (danmainah) [inline index.html:137]: - [x] Kindly make sure to follow the design given for this section, below is a comparison between your design with the given design,


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: html/about.html [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+            <br>
+            <span>
+                <a class="contact-us" href="#">
+                    globaldrumfestival@gdf-portal.org
+                </a>
+            </span>
+        </p>
+
+    </section>
+    <section id="past-global-drum-fest">
```
Comment (danmainah) [inline html/about.html:98]: - [x] Kindly check the section highlighted below, your design is missing one section.
image

File: html/about.html [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+        <h3 class="past-fest-header">See the past Global Drum Festival</h3>
+        <div class="thin-line"></div>
+        <p>Take a look at the last two festival which took place in Gasglow ans Amsterdam</p>
+        <div class="past-image-courosal">
+
+        </div>
```
Comment (danmainah) [inline html/about.html:104]: - [x] Kindly reduce the number of images to match the design given.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+            </button>
+            <div class="hamburger-btn">
+                <img class="hamburger" src="./assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
+        </nav>
+    </header>
+    <div class="back-drop">
+        <div class="faded-backdrop"></div>
+        <div class="headline-bg-image"></div>
+        <!-- <img src="./assets/images/DSC_1494-edited-1.jpg" alt="Chris Coleman"> -->
```
Comment (danmainah) [inline index.html]: - [x] Kindly remove this commented line, it's not good practice to have commented code in your file.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @bobb-Rob , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️

- [ ] Kindly make sure to include a video description, presenting the features of your project,this is the project requirement.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by bobb-Rob** [comment]
The video describing my project is attached in the Readme file.

https://www.loom.com/share/fb7f9611b25f4b13bbc9d0eab6e2fdb2

File: README.md (hunk 1 above)
Comment (zieeco) [reply README.md]: - [x] This issue is well resolved :heavy_check_mark:

File: README.md (hunk 2 above)
Comment (zieeco) [reply README.md:14]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 4 above)
Comment (zieeco) [reply index.html:68]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 5 above)
Comment (zieeco) [reply index.html:20]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 6 above)
Comment (zieeco) [reply index.html]: - [x] This issue is well resolved :heavy_check_mark: 
Also, the width is opposed to change on moving to another width :heavy_check_mark:

File: index.html (hunk 7 above)
Comment (zieeco) [reply index.html:137]: - [x] This issue is well resolved :heavy_check_mark: 
You nailed it my fellow drummer :+1: :heavy_check_mark:

File: html/about.html (hunk 8 above)
Comment (zieeco) [reply html/about.html:98]: - [x] This issue is well resolved :heavy_check_mark:

File: html/about.html (hunk 9 above)
Comment (zieeco) [reply html/about.html:104]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 10 above)
Comment (zieeco) [reply index.html]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+            <div class="hamburger-btn">
+                <img class="hamburger" src="./assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
+        </nav>
+    </header>
+
+    <!-- <div class="back-drop">
+        <div class="faded-backdrop"></div>
+        <div class="headline-bg-image"></div>      
+    </div> -->
```
Comment (zieeco) [inline index.html:75]: - [x] **[OPTIONAL]**

- Let's keep our codebase clean by removing unused commented codes.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @bobb-Rob 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
_Check the comment under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 11 above)
Comment (bobb-Rob) [reply index.html:75]: Hi @zieeco , thanks for the code review. I am even more happy you are also a drummer,...nice one. 
I would like to connect with you more.

**Pull request merged**